contracts/campaignAirdrops/NewSB1155DropCampaign.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+
+import "./shared/CampaignAirdropsShared.sol";
+
+/// @title Airdrops new SoulBound ERC1155 token for airdrop recipients
+contract NewSB1155DropCampaign is ERC1155, CampaignAidropsShared {
+    uint256 private constant _tokenId = 0;
+    uint256 private constant _tokenAmount = 1;
+    uint256 public constant tokensPerClaim = 1; // 1 reward per wallet
+    string public constant airdropType = "SB1155";
+
+    uint256 public airdropFundBlockTimestamp;
+
+    address internal airdropFundingHolder;
+
+    error SoulboundTokenUntransferable();
+    error NotTokenOwner();
+
+    event Attest(address indexed to);
+    event Revoke(address indexed from);
+
+    constructor(string memory uri, address _airbroCampaignFactoryAddress)
+        ERC1155(uri)
+        CampaignAidropsShared(_airbroCampaignFactoryAddress)
+    {}
+
+    /// @notice Sets the merkleRoot - can only be done if admin (different from the contract owner)
+    /// @dev Implements a handler method from the parent contract for performing checks and changing state
+    /// @param _merkleRoot - The root hash of the Merle Tree
+    function setMerkleRoot(bytes32 _merkleRoot) external onlyAdmin {
+        super.setMerkleRootHandler(_merkleRoot);
+
+        emit MerkleRootSet(_merkleRoot);
```

Comment:
This event should be inside of paremt method, and then this method doesn't need to exist in a child contract